tests/integration/04.Edge.Tests.ps1

```diff
@@ -660,6 +660,67 @@ Describe "Edge" {
         }
     }
 
+    Context "SSH" {
+
+        it "Can disable SSH" {
+            $edge = Get-NsxEdge $name
+            #When deploy pstester ESG, the SSH is enabled
+            $edge.cliSettings.remoteAccess | should be "true"
+            Get-NsxEdge $name | Disable-NsxEdgeSsh -confirm:$false
+            $edge = Get-NsxEdge $name
+            $edge.cliSettings.remoteAccess | should be "false"
+        }
+
+        it "Can enable SSH" {
+            Get-NsxEdge $name | Enable-NsxEdgeSsh
+            $edge = Get-NsxEdge $name
+            $edge.cliSettings.remoteAccess | should be "true"
+        }
+    }
+
+    Context "CliSettings" {
+
+        it "Can disable SSH" {
+            $edge = Get-NsxEdge $name
+            #When deploy pstester ESG, the SSH is enabled
+            $edge.cliSettings.remoteAccess | should be "true"
+            #Need Always to set a password to change cliSettings
+            Get-NsxEdge $name | Get-NsxcliSettings | Set-NsxCliSettings -password Vmware1!Vmware1! -remoteAccess:$false
```

Comment:
you havent tested get-nsxclisettings yet - you need prior test that tests this (and you should use it two lines above too!) - this will probably be a moot point if you move these functions to set-nsxedge though...

Reply:
do you have a example ?

Reply:
all the other tests that test retrieval first.